File: cn/java-spring-demo/src/main/java/com/itangcent/springboot/demo/common/model/TestJsonIgnoreBean.java
package com.itangcent.springboot.demo.common.model;

import com.fasterxml.jackson.annotation.JsonIgnore;

/**
 * JSON字段忽略测试Bean
 * 用于演示Jackson的@JsonIgnore注解功能
 * 在JSON序列化时忽略敏感字段
 */
public class TestJsonIgnoreBean {
    /**
     * 用户名
     * 正常参与JSON序列化
     */
    private String userName;

    /**
     * 密码
     * 使用@JsonIgnore注解，在JSON序列化时忽略此字段
     * 用于保护敏感信息
     */
    @JsonIgnore
    private String password;

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
} 

File: en/kotlin-spring-demo/src/main/kotlin/com/itangcent/common/model/TestJsonIgnoreBean.kt
package com.itangcent.common.model

import com.fasterxml.jackson.annotation.JsonIgnore

/**
 * JSON Field Ignore Test Bean
 * Used to demonstrate Jackson's @JsonIgnore annotation functionality
 * Ignores sensitive fields during JSON serialization
 */
data class TestJsonIgnoreBean(
    /**
     * Username
     * Normal JSON serialization
     */
    var userName: String? = null,

    /**
     * Password
     * Using @JsonIgnore annotation, ignored during JSON serialization
     * Used to protect sensitive information
     */
    @JsonIgnore
    var password: String? = null
) 